crates/ruff_python_resolver/src/lib.rs

```diff
@@ -129,6 +128,14 @@ mod tests {
         env_logger::builder().is_test(true).try_init().ok();
     }
 
+    macro_rules! assert_debug_snapshot_normalize_paths {
+        ($value: ident) => {
+            // The debug representation for the backslash are two backslashes (escaping)
+            let $value = format!("{:#?}", $value).replace("\\\\", "/");
```

Comment:
Nitnitnit: What was it again about macro clashes ;)

Reply:
haha, true, that issue also got fixed really quickly